### PR TITLE
Add Child Spans for Interactive Span

### PR DESCRIPTION
**Pull request by tanmaymanolkar1**
Helper PR for : https://github.com/AzureAD/ad-accounts-for-android/pull/2970

### CODE REVIEW

File: common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java [hunk 1]
```diff
@@ -33,6 +33,8 @@ public enum SpanName {
     GetAccounts,
     RemoveAccount,
     WorkplaceJoin,
+    preInterruptInteractive,
+    preInterruptSilent,
```
Comment (mohitc1) [inline common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java]: These names don't necessarily reflect their what they are being used for.

We should have better names

Comment (tanmaymanolkar1) [reply common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java]: @mohitc1 any suggestions?

Comment (tanmaymanolkar1) [reply common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java]: renamed it to ATIInteractively and ATISilently,

Comment (mohitc1) [reply common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java]: It would be good to add comment describing what it's used for.

**Review by mohitc1 — APPROVED** [review]
:shipit:

**Review by shahzaibj — APPROVED** [review]
Not a fan of these names but I guess this is fine

**Pull request merged**